airgun/views/subscription.py

```diff
@@ -186,14 +202,21 @@ class SubscriptionDetailsView(BaseLoggedInView):
     @View.nested
     class details(SatTab):
 
+        associations = SatTable(
```

Comment:
Are you sure 'associations' is better than our standard 'table'? Or maybe there're other tables and no search for sure? Pls explain your choice.

Reply:
I really do not understand what you mean , but associations here is a table in subscription details page:
![screenshot from 2018-11-22 11-33-48-sub-details](https://user-images.githubusercontent.com/131553/48894212-d4cd1200-ee4a-11e8-836f-2f9a6f417f8d.png)

Reply:
Ok, great, i thought it was regular table with entities in it. For such table custom name makes sense, thx.